src/project/FileTreeView.js

```diff
@@ -59,6 +59,8 @@ define(function (require, exports, module) {
     var CLICK_RENAME_MINIMUM  = 500,
         RIGHT_MOUSE_BUTTON    = 2,
         LEFT_MOUSE_BUTTON     = 0;
+    
+    var INDENTATION_WIDTH     = 18;
```

Comment:
As it turns out in #11717, the indentation should only be 10px, and now that it's merged, that's the behaviour on master, too. I think it looks way cleaner.